src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -24,29 +42,121 @@ import {CdkMenuItem} from './menu-item';
   exportAs: 'cdkMenuTriggerFor',
   host: {
     'aria-haspopup': 'menu',
+    '[attr.aria-expanded]': 'isSubmenuOpen() || null',
```

Comment:
So, opposite of other attributes, I think we *do* want `aria-expanded` to always be present because it cues the user that there's something here to be expanded (otherwise it seems like a regular button)